src/utils/constants.ts

```diff
@@ -12,3 +12,8 @@ export const STANDARD_TIMEOUT = 1000;
 export const INTERMEDIATE_TIMEOUT = 3000;
 export const MAX_RETRY_DELAY = 15_000;
 export const Error500 = 500;
+// Upload max file size limit
+export const FILE_SIZE_LIMIT = 2e9; // 2 GB - 2_147_483_648
+export const FILE_SIZE_LIMIT_TEST = 2e6; // 2MB - 2_097_152
```

Comment:
Might be nice to simply include the sizes in the var names to help clarify the code that uses them.

```suggestion
export const FILE_SIZE_LIMIT_2GB = 2e9; // 2 GB - 2_147_483_648
export const FILE_SIZE_LIMIT_2MB = 2e6; // 2MB - 2_097_152
```